README.md

```diff
@@ -1 +1,77 @@
-# hello-microverse-project
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+# 📖 [Hello Microverse project]
+
+**[Hello Microverse project]** is a simple project that display a hello microverse message.
+
+## 🛠 Built With
+
+### Tech Stack
+
+**Client**
+React.js
+**Server**
+Express.js
+**Database**
+PostgreSQL
+
```

Comment:
[OPTIONAL] Great section to have 😉 but how about it reflects your project and the stack you really used? Also, make sure to use the right markdown format. Keep going ✨